src/pathfinder/aystar.h

```diff
@@ -92,45 +46,66 @@ typedef void AyStar_FoundEndNode(AyStar *aystar, PathNode *current);
  * The #user_path, #user_target, and #user_data[10] are intended to be used by the user routines. The data not accessed by the #AyStar code itself.
  * The user routines can change any moment they like.
```

Comment:
I think most of these comment lines should be removed, because the variables don't exist any more.